.slashrc

```diff
@@ -83,6 +83,7 @@ class MediaPlugin(slash.plugins.PluginInterface):
       help = "number of call timeouts allowed per run")
     parser.add_argument("--parallel-metrics", action = "store_true")
     parser.add_argument("--platform", default = None, type = str, required = True)
+    parser.add_argument("--device", default = '/dev/dri/renderD128', type = str, help = 'ffmpeg run with render device')
```

Comment:
Maybe we should have this option apply to gstreamer, too?  For gstreamer, we can set the appropriate environment variable to this parameter.

https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/-/merge_requests/1953
https://gitlab.freedesktop.org/gstreamer/gstreamer-vaapi/-/merge_requests/409